tests/components/application-connector/resources/charts/gateway-test/charts/test/templates/applications/proxy-cases.yml

```diff
@@ -36,3 +36,21 @@ spec:
         - type: API
           targetUrl: "http://{{ .Values.global.mockServiceName }}.{{ .Values.global.namespace  }}.svc.cluster.local:8080/v1/api/unsecure/code/203"
           centralGatewayUrl: "http://central-application-gateway.kyma-system:8080/proxy-cases/code-203"
+    - displayName: code 500
```

Comment:
Could you consider adding this test?

https://github.com/kyma-project/kyma/blob/7ff0070341f136ea885d8c66efec4cc896fac3ea/tests/components/application-connector/resources/charts/gateway-test/charts/test/templates/applications/proxy-errors.yml#L12-L21

It was removed due to incompatibility of Golang and http package when proxying 1xx requests. If the problems persist you can hold off on adding it.